app/components/Progress.tsx

```diff
@@ -0,0 +1,13 @@
+export function Progress({ progress }: { progress: number }) {
```

Comment:
Radix has a [progress component](https://www.radix-ui.com/docs/primitives/components/progress) we may want to use.

Reply:
Yeah I attempted using that but was having a issues